R/constants.R

```diff
@@ -31,6 +32,14 @@ GTAP_HISTORICAL_YEAR <- 2000
 aglu.GLU <- "GLU"
 aglu.GLU_NAME_DELIMITER <- ""  # delimiter between the GLU name and number
 
+# cotton seed and lint conversion
+CONV_COTTON_LINT <- 0.4
+
+# Price conversion from alfalfa to grass hay
+PRICERATIO_GRASS_ALFALFA <- 0.7
```

Comment:
Here are the sources for the fodder price data, and the conversion from alfalfa prices to grass prices:
Alfalfa price: USDA. 2011b. Prices Received for Alfalfa Hay, Baled, Washington. National Agricultural Statistics Service,  U.S. Department of Agriculture.
Grass price: Baker, A., and H. Lutman. 2008. Feed Year in Review (Domestic): Record Demand Drives U.S. Feed Grain Prices Higher in 2007/2008. FDS-2008-01, Economic Research Service, United States Department of Agriculture. Available at http://usda.mannlib.cornell.edu/usda/ers/FDS-yearbook/2000s/2008/FDS-yearbook-05-23-2008_Special_Report.pdf